api/app/controllers/spree/api/v2/swagger_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Spree
```

Comment:
@bbonislawski let's move API preview to a separate gem and leave here only the swagger contracts